src/main/java/org/jullaene/walkmong_back/api/apply/service/ApplyService.java

```diff
@@ -1,11 +1,33 @@
 package org.jullaene.walkmong_back.api.apply.service;
 
 import lombok.RequiredArgsConstructor;
+import org.jullaene.walkmong_back.api.apply.domain.Apply;
+import org.jullaene.walkmong_back.api.apply.dto.req.ApplyRequestDto;
 import org.jullaene.walkmong_back.api.apply.repository.ApplyRepository;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @RequiredArgsConstructor
 public class ApplyService {
     private final ApplyRepository applyRepository;
+
+    @Transactional
+    public Long saveApply(ApplyRequestDto applyRequestDto){
+        Apply apply=Apply.builder()
+                .dongAddress(applyRequestDto.getDongAddress())
+                .roadAddress(applyRequestDto.getRoadAddress())
+                .latitude(applyRequestDto.getLatitude())
+                .longitude(applyRequestDto.getLongitude())
+                .addressDetail(applyRequestDto.getAddressDetail())
+                .addressMemo(applyRequestDto.getAddressMemo())
+                .poopBagYn(applyRequestDto.getPoopBagYn())
+                .muzzleYn(applyRequestDto.getMuzzleYn())
+                .dogCollarYn(applyRequestDto.getDogCollarYn())
+                .preMeetingYn(applyRequestDto.getPreMeetingYn())
+                .memoToOwner(applyRequestDto.getMemoToOwner())
+                .build();
```

Comment:
```suggestion
// in ApplyService
        Apply apply=Apply.builder()
                .applyRequestDto(applyRequestDto)
                .build();
                
// in Apply
@Builer()
public Apply (ApplyRequestDto applyRequestDto) {
                dongAddress = applyRequestDto.getDongAddress());
                // 나머지들
                }
```
dto를 entity로 변환할 때에는 생성자에 인자로 받아서 저장하는 게 좋을 것 같습니다
@Getter는 최대한 dto에서 사용할 수 있도록 해주세욤